types/rockset/rockset-tests.ts

```diff
@@ -0,0 +1,5 @@
+import { Collection } from "rockset";
+
+function rockset_tests() {
+    // TODO
```

Comment:
Could you please add a couple of tests before merging the pull request?

Reply:
sure, we had a user waiting for the definitions so created a pull request without tests. Will add some.

Reply:
Added some tests